saleor/core/utils/metadata_manager.py

```diff
@@ -73,3 +74,12 @@ def create_from_graphql_input(
     return MetadataItemCollection(
         [MetadataItem(item.key, item.value) for item in items]
     )
+
+
+def method_metadata_is_valid(metadata: Any) -> bool:
```

Comment:
missing test

Reply:
This part is just moved from other place. But I will add tests

Reply:
done